articles/send-sms-in-nodejs-using-vonage-api/index.md

```diff
@@ -0,0 +1,229 @@
+Short Message Service (SMS) is one of the best marketing strategies that organizations can use to keep their clients close.  SMS integration in a mobile or a web application effectively reaches out to customers as most people check their message inbox than emails. SMS are mobile-friendly and can reach a broad demographic cover.
+Vonage's SMS API allows programmers to integrated messaging services into an application with ease. Vonage's API's benefits are support local phone numbers, low latency, high delivery rates, and modern web technologies support.
```

Comment:
Vonage's SMS API allows programmers to `integrated` messaging services into an application with ease

integrate